LeaveManagementModule/Frontend/admin/admin.html

```diff
@@ -0,0 +1,237 @@
+                </div>
+            </div>
+        
+        
+        
+            <div class="modal fade" id="deleteModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+                <div class="modal-dialog" role="document">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h5 class="modal-title" id="exampleModalLabel">Delete</h5>
+                            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                                <span aria-hidden="true">&times;</span>
+                            </button>
+                        </div>
+                        <div class="modal-body">
+                            <form class="leave-request-form">
+                                <div class="form-group">
+                                    <label for="recipient-name" class="col-form-label">Are you sure you want to delete ?</label>
+                                    
+                                </div>
+                               
+                            </form>
+                        </div>
+                        <div class="modal-footer">
+                            <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                            <button type="button" class="btn btn-primary">Confirm</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            
+        <!-- Optional JavaScript -->
+
+        <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+        <script
+                src="https://code.jquery.com/jquery-3.3.1.min.js"
+                integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+                crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+
+        <script>
+           var data = [{ "type":'Casual Leave',"total" : '10', "MaxConsLeave" : '4', "Edit":'<button type="button" class="btn btn-success" data-toggle="modal" data-target="#editModal"> Edit </button>',"Delete":'<button type="button" class="btn btn-danger" data-toggle="modal" data-target="#deleteModal">Delete</button>'},
+                       { "type":'Sick Leave',"total" : '10', "MaxConsLeave" : '4', "Edit":'<button type="button" class="btn btn-success" data-toggle="modal" data-target="#editModal"> Edit </button>',"Delete":'<button type="button" class="btn btn-danger" data-toggle="modal" data-target="#deleteModal">Delete</button>'},
```

Comment:
button tag should not come in data . this variable represent data which will be sent by server on making a call to api